lib/rubocop/cop/style/inverse_methods.rb

```diff
@@ -76,6 +78,7 @@ def on_block(node)
           inverse_block?(node) do |_method_call, method, block|
             return unless inverse_blocks.key?(method)
             return if negated?(node) && negated?(node.parent)
+            return if node.each_node(:next).any?
```

Comment:
> It was sufficient to consider `next` nodes in testing this over a large codebase, but it also guards agains `break` and `return` nodes, out of an abundance of caution.

I don't see any code in here that would fix this issue for `break` or `return`. I think this needs to look for those node types as well.

Reply:
@rrosenblum it's been reverted after further discussion/consideration. could i trouble you for an example of an issue caused by `break` or `return`?

Reply:
I think I missed something from an earlier conversation. It sounds like you had accounted for this and then reverted it. The comment that I quoted made it seem like we needed to account for `break` and `return`. These are likely edge cases because `break` and `return` don't make much sense inside of methods that are designed to have a return.